src/apps/common/submodules/portListing/portListing.js

```diff
@@ -504,6 +507,25 @@ define(function(require) {
 			template.find('#startDate').datepicker('setDate', fromDate);
 			template.find('#endDate').datepicker('setDate', toDate);
 
+			template
+				.find('.port-filter')
+					.on('click', function(event) {
+						event.preventDefault();
+
+						var fromDate = template.find('input.filter-from').datepicker('getDate'),
+							toDate = template.find('input.filter-to').datepicker('getDate'),
+							tab = self.portListingGet('tab'),
+							type = self.portListingGet('type');
+
+						self.portListingRenderTableList({
+							template: self.portListingGet('container'),
+							fromDate: fromDate,
+							toDate: toDate,
+							type: type,
+							tab: tab
+						});
+					});
+
```

Comment:
Curious why you moved this? AFAICT there is no changes?

It is okay move code around but you should create a specific commit with a message explaining your intent so PR reviews do not turn into "where is waldo" puzzles.

Reply:
This is what was causing the weired issue, this event was binding with the table pagination/rendering causing the extra click events, but this element (filter button) isn't part of the table, this is why I moved it to the correct binding method.

My bad, I will create a different commit for future cases, thanks for the suggestion